### ISSUE TITLE
Custom keymappings in `settings.lua` are ignored.

### ISSUE THREAD

**Issue by Rot127** [kabinspace/AstroVim#213]
I try to add some vim key bindings in the `settings.lua` file but they are ignored.

```lua
-- Move lines and blocks
map("n", "<C-K>", "<cmd>move .+1<CR>==", opts)
map("n", "<C-J>", "<cmd>move .-2<CR>==", opts)
map("v", "<C-K>", "<cmd>move '>+1<CR>gv=gv", opts)
map("v", "<C-J>", "<cmd>move '<-2<CR>gv=gv", opts)
```

Is this a bug? Or am I missing something?

**Comment by mehalter** [comment]
are you putting this in the `settings.lua` or have you migrated to the new configuration for `v1.0`? These should be located now inside of `user/init.lua` within the `polish` function. And example is shown in the `user_example` folder on the repo

https://github.com/kabinspace/AstroVim/blob/a894214df49b96f774a1d1ecd6531213721b86a0/lua/user_example/init.lua#L145-L154

**Comment by Rot127** [comment]
> are you putting this in the settings.lua or have you migrated to the new configuration for v1.0

Yes, `settings.lua`. I can't update to v1.0 yet because of https://github.com/kabinspace/AstroVim/issues/162 (updating to nvim 7.0 dev is not an option for me).

Guess I have to wait then until nvim 7.0 is on the release page.

**Comment by mehalter** [comment]
Hm, you should give it a try. That is the only time I have seen nvim 0.7 be a limitation. I also don't use nvim 0.7.

The bindings still should go in the `polish` function so the same as the `init.lua` file thta I linked but you would name it `settings.lua`.

**Comment by mehalter** [comment]
@Rot127 were you able to resolve this issue?

**Comment by Rot127** [comment]
I get build errors on the neovim main and don't want to fix them since nvim 7.0 is released in a few weeks.
Will reopen if the error still exists with nvim7.0
